origin-contracts/package-lock.json

```diff
@@ -10,16 +10,16 @@
 			"integrity": "sha512-zbO/DtTnaDappBflIU3zYEgATLToRDmW5uN/EGH1GXaes7ydfjqmAoK++xmJIA+8HfDw7UyPZNdM8fhGhfmMhw==",
 			"dev": true,
 			"requires": {
-				"chokidar": "^2.0.3",
-				"commander": "^2.8.1",
-				"convert-source-map": "^1.1.0",
-				"fs-readdir-recursive": "^1.1.0",
-				"glob": "^7.0.0",
-				"lodash": "^4.17.10",
-				"mkdirp": "^0.5.1",
-				"output-file-sync": "^2.0.0",
-				"slash": "^2.0.0",
-				"source-map": "^0.5.0"
+				"chokidar": "2.0.4",
```

Comment:
Any reason to commit the package-lock in this PR?

Reply:
Don't think so - I'll pull it out